RealestateApi/Dto/Response/AddCustomDtoResponse.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace RealEstateApi.Dto.Response
+{
+    public class AddCustomDtoResponse
```

Comment:
This DTO isn't needed because we can use CustomerModel instead of AddCustomDtoResponse

Reply:
Done